src/pages/index.tsx

```diff
@@ -1,17 +1,59 @@
+import HeadingDescription from '../components/HeadingDescription';
+import Button from '../components/Buttons';
+
 export default function Home() {
   return (
     <>
       <section>
         <h1>Learning and teaching online, made easy.</h1>
       </section>
       <section>
-        <h2>Learn a language in a playful way</h2>
-      </section>
-      <section>
-        <h2>Increase your vocabulary</h2>
-      </section>
-      <section>
-        <h2>Watch your progress everyday</h2>
+        <section className="languageSectionContainer">
```

Comment:
I recommend refactoring the sections into separate components for better code organization and reusability. Additionally, the vocabularySectionContainer and statisticsSectionContainer appear to have the same structure. Consider refactoring them into a single component to reduce duplication and improve maintainability.

Reply:
I have already extracted sections from main page to the OtherSections component.

Regard to the current code, I did think about that not even vocabularySectionContainer and statisticsSectionContainer, but also VocabularySectionContainer, which three other sections can form as one component.

But I'd like to wait to the next refactoring sprint, as the possible responsive may be added on, which may potential increase the props number passing to the component.